test/Auto/build.sh

```diff
@@ -0,0 +1,55 @@
+echo "Clone repo start"
+sudo git clone -b Auto https://github.com/zhichengzhang1995/f5-adcaas-openstack.git
+echo "Apt update start"
+sudo apt-get update
+echo "Nodejs install start"
+# sudo apt-get install nodejs -y
+echo "Npm install start"
+# sudo apt-get install npm -y
+echo "Docker install start"
+# sudo apt-get install apt-transport-https ca-certificates curl gnupg-agent     software-properties-common -y
+# sudo apt install docker.io -y
+echo "Typescript install start"
+# sudo npm install -g typescript -y
+# echo yes | sudo apt install python-pip -y
+echo "Docker compose install start"
+# sudo curl -L "https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+# sudo chmod +x /usr/local/bin/docker-compose
+
+sudo npm install --save-dev eslint-config-prettier@^5.0.0
+# sudo npm install --save-dev eslint@^1.1.2
+sudo systemctl start docker
+
+echo "App build and install start"
+cd f5-adcaas-openstack/app/waf
+
+echo "Npm install start"
+sudo npm install
+echo "Npm build start"
+sudo npm run build
+echo "Portal install start"
+sudo docker build . -t f5devcentral/f5-openstack-services:portal-latest
+echo "Latest install start"
+sudo docker build . -t f5devcentral/f5-openstack-services:adcaas-latest
+
+echo "Start app"
+cd ../../
+
+# install rpm
+sudo mkdir -p dependencies/ASGExtensions/
+echo "Move rpm start"
+sudo mv ../f5-appsvcs-3.10.0-5.noarch.rpm ./dependencies/
+sudo mv ../f5-declarative-onboarding-1.5.0-11.noarch.rpm ./dependencies/
+sudo mv ../TrustedASMPolicies-1.0.5-0004.noarch.rpm ./dependencies/ASGExtensions/
+sudo mv ../TrustedDevices-1.3.0-0002.noarch.rpm ./dependencies/ASGExtensions/
+sudo mv ../TrustedExtensions-1.0.1-0001.noarch.rpm ./dependencies/ASGExtensions/
+sudo mv ../TrustedProxy-1.0.1-0002.noarch.rpm ./dependencies/ASGExtensions/
+
+cd scripts
+echo "Start test"
+sudo ./start_all.sh
+
+echo "Test app"
+cd ../test
+# sudo npm install -g newman newman-reporter-html
+sudo ./test_all.sh
```

Comment:
Your pr will not need to guarantee test suite must success. I suggest you can skip the test suite. Just provision the environment, start the apps, and make sure the apps are running.

Reply:
We can add new steps to run test suite in the future.

Reply:
fixed.